lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/SraToBam.pm

```diff
@@ -288,7 +292,9 @@ sub convert_fastq_to_bam {
         my $fastq_to_sam = Genome::Model::Tools::Picard::FastqToSam->create(
             fastq           => "$unaligned_fastq",
             output          => "$unaligned_bam",
-            sample_name     => "$sample_name",
+            sample_name => $self->sample_name,
+            library_name => $self->library_name,
+            read_group_name => $self->library_name,
```

Comment:
I see that this class doesn't have a `$self->read_group_name`, so I'm assuming the re-use of `library_name` is intentional here?